### PR TITLE
Ds3/cogniance angular directives

### PR DESCRIPTION
**Pull request by gyerzh**

### CODE REVIEW

File: app/core/data/navigation.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
               "value": "foundation.typography"
             }, {
               "label": "Angular Directives",
-              "value": "foundation.directives"
+              "value": "directives"
```
Comment (gyerzh) [inline app/core/data/navigation.json]: please note that I move directives to own module
I did it for consistency with Web->Elements 

if it's not ok and we need foundation module that combines several features(like angular-directives, animation, color, etc) please let me know

File: app/index.html [hunk 2]
```diff
@@ -232,6 +232,13 @@
     <script src="modules/charts/controllers/charts-morris.controller.js"></script>
     <script src="modules/charts/controllers/charts.controller.js"></script>
 
+    <!-- directives -->
+    <script src="modules/angular-directives/directives.module.js"></script>
+    <script src="modules/angular-directives/directives.routes.js"></script>
+    <script src="modules/angular-directives/directives.controller.js"></script>
```
Comment (gyerzh) [inline app/index.html]: Note: also I moved angular-directives to own folder

File: app/modules/angular-directives/directives.controller.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+(function () {
+  'use strict';
+
+  angular.module('pb.ds.directives').controller('DirectivesController', function ($log) {
```
Comment (gyerzh) [inline app/modules/angular-directives/directives.controller.js]: Note: own module

File: .gitignore [hunk 4]
```diff
@@ -10,8 +10,11 @@ npm-debug.log
 scss-lint.log
 .idea
 build
+dist
```
Comment (gyerzh) [inline .gitignore]: guess dist folder is not needed to commit

Comment (smlombardi) [inline .gitignore]: It should be

Comment (gyerzh) [inline .gitignore]: I removed commit with this change

**Pull request merged**